core/ledger/kvledger/tests/env.go

```diff
@@ -159,6 +159,24 @@ func (e *env) verifyRebuilableDoesNotExist(flags rebuildable) {
 	}
 }
```

Comment:
are these two functions still being used somewhere?

Reply:
Removed. It was used by env.closeAllLedgersAndDrop. Per our slack, changed the func to remove contents instead of deleting the directory itself.